### PR TITLE
Smoke Test failure - Name fix for Runtime template

### PR DESCRIPTION
**Pull request by Raghul-M**

### CODE REVIEW

**Review by ConorOM1 — DISMISSED** [review]

File: ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot [hunk 1]
```diff
@@ -5,14 +5,15 @@ Resource          ../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRunt
 Resource          ../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
 Resource          ../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
 Suite Setup       Custom Serving Runtime Suite Setup
-Suite Teardown    Custom Serving Runtime Suite Teardown
+#Suite Teardown    Custom Serving Runtime Suite Teardown
```
Comment (tarukumar) [inline ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot:8]: Why teardown is commented ?

Comment (Raghul-M) [reply ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot:8]: I forgot to uncomment it , let me uncomment it now

**Review by ConorOM1 — DISMISSED** [review]

File: ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot [hunk 2]
```diff
@@ -25,7 +26,7 @@ Verify RHODS Admins Can Import A Custom Serving Runtime Template By Uploading A
     Open Dashboard Settings    settings_page=Serving runtimes
     Upload Serving Runtime Template    runtime_filepath=${OVMS_RUNTIME_FILEPATH}
     ...    serving_platform=multi      runtime_protocol=gRPC
-    Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_DISPLAYED_NAME}
+    Serving Runtime Template Should Be Listed    displayed_name=${UPLOADED_OVMS_XPATH_NAME}
```
Comment (tarukumar) [inline ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot:29]: Where is the `displayed_name=${UPLOADED_OVMS_XPATH_NAME` varible name i f9 not see in the suite

Comment (Raghul-M) [reply ods_ci/tests/Tests/1000__model_serving/1003__model_serving_customruntimes.robot:29]: fixed it

**Review by tarukumar — APPROVED** [review]

**Review by ConorOM1 — APPROVED** [review]

**Pull request merged**